send/ldap_it4i

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/perl
+use strict;
+use warnings;
+no if $] >= 5.017011, warnings => 'experimental::smartmatch';
+use Net::LDAPS;
+use Net::LDAP::Entry;
+use Net::LDAP::Message;
+use Net::LDAP::LDIF;
+use ScriptLock;
```

Comment:
You should add `use ADConnector;` if you are using this library methods.

Reply:
Added.